dvc/remote/azure.py

```diff
@@ -57,8 +58,23 @@ def __init__(self, repo, config):
             or os.getenv("AZURE_STORAGE_CONTAINER_NAME")
         )
 
+        # read custom environment variables
+        connection_string_from_config = config.get(
```

Comment:
Would recommend extracting this whole code chunk to method, like
`connection_string_from_config = _read_connection_string_from_config(...)`